test/integration/targets/pids/tasks/main.yml

```diff
@@ -0,0 +1,82 @@
+# Test code for the pid module
+# Copyright 2018, Saranya sridharan
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- name: "Installing PidOf"
+  yum:
+    name: sysvinit-tools
+    state: present
+  when: ansible_distribution == 'CentOS'
+
+- name:  "Installing PidOf in freebsd"  
+  pkgng:
+    name: pidof
+    state: present
+  when: ansible_distribution == 'FreeBSD'
+  
+- name: Install  PidOf in OSX
+  block:
+    - name: Find brew binary
+      command: which brew
+      register: brew_which
+    - name: Get owner of brew binary
+      stat: path="{{ brew_which.stdout }}"
+      register: brew_stat
+    - name: "Install package"
+      homebrew:
+        name: pidof
+        state: present
+        update_homebrew: no
+      become: yes
+      become_user: "{{ brew_stat.stat.pw_name }}"
+      
+  when: ansible_os_family == 'Darwin'
+
+- name: "Installing setproctitle"
+  pip:
+    name: setproctitle
+    version: 1.1.10
+
+- name: "Checking empty result" 
+  pids: 
+    name: "blahblah"
+  register: emptypids
+   
+- name: "Verify that the list of Process IDs (PIDs) returned is empty"
+  assert:
+    that:
+    - emptypids is not changed
+    - emptypids.pids == [] 
+   
+- name: Executing the python script
+  command: ./sample.py
+  args:
+    chdir: /var/root/ansible/test/integration/targets/pids/tasks
```

Comment:
Can't you use `{{ role_path }}` to make this OS-independent? CC @mattclay

Right now, if the files are ever moved somewhere else, this will break.

Reply:
@saranyasridharan Use of `chdir` shouldn't be needed. Why was this added?

Reply:
Without specifying path, I was not able to execute that. 'No file found error' occured. So I added this.